edk2toollib/uefi/edk2/parsers/base_parser.py

```diff
@@ -43,18 +45,26 @@ def SetBaseAbsPath(self, path):
 
         """
         self.RootPath = path
```

Comment:
what is path?  absolute or relative.  Relative to what (CWD?)

Reply:
alright- converted to an abs path